### ISSUE TITLE
Some functions are listed with the operators

### ISSUE THREAD

**Issue by abrudz** [jpellegrini/gnu-apl-refcard#3]
`A/B`, `A⌿B`, `A\B`, A⍀B` are in fact functions in GNU APL, even though they are operators in APL2.

**Comment by jpellegrini** [comment]
Thanks for your issue reports!
But I am not sure what you mean in this one -- the refcard has a list of "notations", and we did not specify which are operators and which are functions...

**Comment by abrudz** [comment]
True, but all the functions (except `⍎` and `⍕` which are arguably special) are listed consecutively from `+` to `⊤`, then comes various system stuff, and finally all the operators from `op/A` until `A∘C`. While not explicitly labelled as such, this is clearly making a distinction between functions and operators.

**Comment by jpellegrini** [comment]
Hm, right. That makes sense! We'll fix that.

**Comment by luksamuk** [comment]
I get the point that these are functions in GNU APL, but since they use the same symbol and we're not making distinctions between functions and operators, should we really make them far apart?

It seems to me that someone looking for the meaning of `⌿`, for example, would expect everything about it close together.

**Comment by jpellegrini** [comment]
> should we really make them far apart?

Oh. Of course not! Functions and operators together would make more sense, you'r right!

**Comment by abrudz** [comment]
> It seems to me that someone looking for the meaning of `⌿`, for example, would expect everything about it close together.

But the refcard is structured as "functionality → syntax" (as opposed to [Dyalog's refcard](http://docs.dyalog.com/latest/ReferenceCard.pdf)) so you should expect someone looking for a function among the functions, not for a specific symbol.
